src/sentry/receivers/onboarding.py

```diff
@@ -165,7 +166,10 @@ def record_release_received(project, group, event, **kwargs):
 
 @event_processed.connect(weak=False)
 def record_user_context_received(project, group, event, **kwargs):
-    if not event.data.get('sentry.interfaces.User'):
+    user_context = event.data.get('sentry.interfaces.User')
+    if not user_context:
+        return
+    elif isinstance(user_context, six.string_types) and user_context[:3] == 'ip:':
```

Comment:
Pretty confident that's false. It'd still be an object with an `ip_address` field. At this point.